services/galley/src/Galley/Data.hs

```diff
@@ -505,6 +505,10 @@ deleteConversation cid = do
 acceptConnect :: MonadClient m => ConvId -> m ()
 acceptConnect cid = retry x5 $ write Cql.updateConvType (params Quorum (One2OneConv, cid))
 
+-- | We deduce the conversation ID by adding the 4 components of the V4 UUID
+-- together pairwise, and then setting the version bits (v4) and variant bits
+-- (variant 2). This means that we always know what the UUID is for a
+-- one-to-one conversation which hopefully makes them unique.
```

Comment:
1. That fact that we always know what a uuid is doesn't make it unique.  What are you trying to say?
2. You are explaining the algorithm in two places.  I this place was just referencing the haddocs of `addv4` (which hopefully will have a better name, too).

Reply:
Trying to say that I don't believe this `addv4` guarantees that if   `x` and `y` were picked from a uniform distribution that `x +  y` is uniformly distributed. 

If we had picked `XOR` instead of  `+` as the combining operator, I'd be pretty confident we maintain this invariant

Reply:
i've understood that much, what i'm trying to say that the logical structure of your sentence is off.  the fact that we know what the UUID is doesn't make it unique, not even hopefully.

too nit-picky?